### PR TITLE
Feature/issue 14

### PR DESCRIPTION
**Pull request by landreaubis**

### CODE REVIEW

File: src/AppBundle/DataFixtures/ORM/bluray.yml [hunk 1]
```diff
@@ -5,4 +5,4 @@ AppBundle\Entity\Bluray:
         type: <word()>
         trailer: Z6Z1f4Lt01o
         available: 1
-        picture: <picture()>
+        picture: <picture()>
```
Comment (Jean-Beru) [inline src/AppBundle/DataFixtures/ORM/bluray.yml]: It's recommended to keep a blank line at the end of each file (see http://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file)

See also https://github.com/FriendsOfPHP/PHP-CS-Fixer to automatically fix your code.

**Comment by Jean-Beru** [comment]
:+1: (with squash :wink: )

File: app/Resources/views/bluray/detail.html.twig [hunk 2]
```diff
@@ -10,6 +10,6 @@
     </div>
     <div class="detailTrailer">
         <h2>Trailer</h2>
-        {{bluray.trailer}}
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/{{bluray.trailer}}" frameborder="0" allowfullscreen></iframe>
     </div>
 {% endblock %}
```
Comment (nlemahieu) [inline app/Resources/views/bluray/detail.html.twig]: Missing empty line!

File: src/AppBundle/DataFixtures/ORM/bluray.yml (hunk 1 above)
Comment (nlemahieu) [inline src/AppBundle/DataFixtures/ORM/bluray.yml]: Same here

**Comment by landreaubis** [comment]
I add this empty line tonight and commit/merge after.

Thx guys ;)

**Pull request merged**